test/FastTests/Client/QueriesWithVariables.cs

```diff
@@ -13,7 +13,7 @@ public QueriesWithVariables(ITestOutputHelper output) : base(output)
         {
         }
 
-        [RavenTheory(RavenTestCategory.Querying)]
+        [RavenTheory(RavenTestCategory.ClientApi | RavenTestCategory.Querying)]
```

Comment:
I'm worried that now each single test will have ClientApi?

Reply:
I was too, but looking at those it actually makes sense... Like you have Querying where you are testing the low level querying mechanism and there are Querying that are also exercising the Client API. If we choose say eventually 'select' just the Querying, why shouldnt we also include the Client API querying part too?